cmd/virt-handler/Dockerfile

```diff
@@ -20,9 +20,18 @@ FROM fedora:26
 
 MAINTAINER "The KubeVirt Project" <kubevirt-dev@googlegroups.com>
 
-RUN dnf -y install libvirt-client \
+RUN dnf -y install libvirt-client cyrus-sasl-scram \
```

Comment:
I believe the scram is the single option if we do not want to over complex authentification with the kerberos or another certification mechanism. Also, virt-handler talk with libvirt via unix socket on the same host, so I believe scram is more than enough.